src/lib.rs

```diff
@@ -77,9 +97,33 @@ module! {
         pub struct GetTransactionByHash as "eth_getTransactionByHash"
             (Digest,) => Option<SignedTransaction>;
 
+        /// Returns the value from a storage position at a given address.
+        pub struct GetTransactionCount as "eth_getTransactionCount"
+            (Address, Option<BlockSpec>) => U256;
+
         /// Returns the receipt of a transaction by transaction hash.
         pub struct GetTransactionReceipt as "eth_getTransactionReceipt"
             (Digest,) => Option<TransactionReceipt>;
+
+        /// Returns the number of uncles in a block from a block matching the given block hash.
+        pub struct GetUncleCountByBlockHash as "eth_getUncleCountByBlockHash"
+            (Digest,) => Option<U256>;
+
+        /// Returns the number of uncles in a block from a block matching the given block number.
+        pub struct GetUncleCountByBlockNumber as "eth_getUncleCountByBlockNumber"
+            (BlockSpec,) => Option<U256>;
+
+        /// Returns the current maxPriorityFeePerGas per gas in wei.
+        pub struct MaxPriorityFeePerGas as "eth_maxPriorityFeePerGas"
+            Empty => U256;
+
+        /// Creates a filter in the node, to notify when a new block arrives.
+        pub struct NewBlockFilter as "eth_newBlockFilter"
+            Empty => U256;
+
+        /// Creates a filter in the node, to notify when new pending transactions arrive.
+        pub struct NewPendingTransactionFilter as "eth_newPendingTransactionFilter"
+            Empty => U256;
```

Comment:
I’m surprised you couldn’t also already do `eth_newLogFilter` since we already have the filter type if I’m not mistaken (used by `eth_getLogs`).

Don’t need to add it to the PR just mentioning there might be a few more “easy” ones that you can add already.

Reply:
Ya it's possible I could have, just wanted to pin down the particularly "easy" ones.